pynubank/utils/discovery.py

```diff
@@ -33,3 +33,7 @@ def _get_url(self, name: str, target: dict) -> str:
             raise NuException(f'There is no URL discovered for {name}')
 
         return url
+
+    def is_alive(client=HttpClient()) -> bool:
```

Comment:
Será que não faria mais sentido essa função ficar num arquivo separado ? Não está diretamente relacionada com essa classe, e sim com todo o pacote do Nubank.

Reply:
Se fizer isso, seria interessante expor essa função no `__init__.py` da raiz, assim quem for usar importa com `from pynubank import is_alive`